iina/PrefSubViewController.swift

```diff
@@ -110,6 +110,8 @@ class PrefSubViewController: NSViewController {
   
   @IBAction func changeDefaultEncoding(_ sender: NSPopUpButton) {
     UserDefaults.standard.set(sender.selectedItem?.representedObject, forKey: Preference.Key.defaultEncoding)
+    PlayerCore.shared.setSubEncoding((sender.selectedItem?.representedObject as? String) ?? "auto")
```

Comment:
Changing default encoding should not effect current encoding. It's an intended behavior.